app/src/main/cpp/skyline/audio/track.h

```diff
@@ -13,14 +16,14 @@ namespace skyline::audio {
     class AudioTrack {
       private:
         const std::function<void()> releaseCallback; //!< Callback called when a buffer has been played
-        std::deque<BufferIdentifier> identifierQueue; //!< Queue of all appended buffer identifiers
+        std::deque<BufferIdentifier> identifiers; //!< Queue of all appended buffer identifiers
 
-        int channelCount; //!< The amount channels present in the track
-        int sampleRate; //!< The sample rate of the track
+        const u8 channelCount; //!< The amount channels present in the track
+        const u32 sampleRate; //!< The sample rate of the track
 
       public:
-        std::queue<i16> sampleQueue; //!< Queue of all appended buffer data
-        skyline::Mutex bufferLock; //!< Buffer access lock
+        CircularBuffer<i16, constant::SampleRate * constant::ChannelCount * 10> samples; //!< A vector of all appended audio samples
```

Comment:
maybe make the '10' a constant here

Reply:
I disagree, it's automatically a constant by being there and it isn't used from anywhere else. I think it's fine here and it's fairly self-explanatory what it is.